--- tests.sql	2017-10-15 14:48:31.000000000 +0000
+++ tests.sql.new	2017-10-15 17:37:38.178687472 +0000
@@ -44,6 +44,6 @@
 	THEN 'Test 11: Failed.' 
 	ELSE 'Test 11: OK.' END;
 
-select CASE WHEN  ST_Buffer(geomfromtext('POINT (2 2)',4326),10,1) != geomfromtext('POLYGON((12 2,2.000000000000016 -8,-8 1.9999999999999676,1.9999999999999538 12,12 2))',4326)
+select CASE WHEN  ST_Buffer(geomfromtext('POINT(2 2)',4326),10) != geomfromtext('POLYGON((12 2,2.000000000000016 -8,-8 1.9999999999999676,1.9999999999999538 12,12 2))',4326)
 	THEN 'Test 12: Failed.' 
 	ELSE 'Test 12: OK.' END;
